apps/nextjs-example/src/components/transactionFlows/MultiAgent.tsx

```diff
@@ -55,9 +60,9 @@ export function MultiAgent() {
       sender: account.address,
       secondarySignerAddresses: [secondarySigner.accountAddress],
       data: {
-        function: "0x1::coin::transfer",
-        typeArguments: [APTOS_COIN],
-        functionArguments: [account.address, 1], // 1 is in Octas
+        bytecode: TRANSFER_SCRIPT,
```

Comment:
whats the reason we use pre compiled script and not an entry function?

Reply:
Since there are no on-chain contract functions that support multi-agent, we have to implement a workaround to make this functionality possible.  
Without this approach, the current multi-agent feature would not be usable.